### ISSUE TITLE
FilteredStreams not working on Tweetinvi 3.0 and above when multiple LanguageFilters are used. 

### ISSUE THREAD

**Issue by jxlarrea** [linvi/tweetinvi#869]
**Reopening #853 because it was closed by mistake by @linvi . This has nothing to do with #850.** This was first reported almost a year ago in #732 

When more than 1 language filters are added to the stream, MatchingTweetReceived is never fired. This happens in Tweetinvi versions 3.0 and above. Version 2.1 Works fine.

In the example below, if you remove the Spanish language filter, the code will work as expected. Otherwise, MatchingTweetReceived is never fired.

```csharp
using System;
using Tweetinvi;
using Tweetinvi.Models;

namespace Tweetinvi_Test
{
    internal class Program
    {
        private static void Main(string[] consoleArgs)
        {
            var credentials = new TwitterCredentials()
            {
                ConsumerKey = "xxx",
                ConsumerSecret = "xxx",
                AccessToken = "xxx",
                AccessTokenSecret = "xxx"
            };

            var fs = Stream.CreateFilteredStream(credentials);
            fs.StallWarnings = true;
            fs.AddTrack("github");
           
            fs.AddTweetLanguageFilter(LanguageFilter.English);
            fs.AddTweetLanguageFilter(LanguageFilter.Spanish);             

            fs.MatchingTweetReceived += (evnt, args) =>
            {
                Console.WriteLine(args.Tweet);
            };

            fs.KeepAliveReceived += (sender, args) =>
            {
                Console.WriteLine("KEEP ALIVE RECIEVED");
            };

            fs.DisconnectMessageReceived += (sender, args) =>
            {
                Console.WriteLine("DisconnectMessageReceived " + args.DisconnectMessage);
            };

            fs.WarningFallingBehindDetected += (sender, args) =>
            {
                Console.WriteLine("WarningFallingBehindDetected " + args.WarningMessage);
            };

            fs.StreamStopped += (sender, args) =>
            {
                Console.WriteLine("StreamStopped " + args.DisconnectMessage);
            };

            fs.StartStreamMatchingAllConditions();
        }
    }
}
```

**Comment by linvi** [comment]
Oh sorry about that. Will be looking into it.

**Comment by linvi** [comment]
Sorry for the very long delay but this was not a priority in regards to all the work that was there.

**Comment by fightlikeagirl** [comment]
Hello,
I have still this problem. When I set one language there is no problem but when set more than one language , no tweet comes. Can you solve it please @linvi ?

**Comment by linvi** [comment]
@fightlikeagirl This bug is fixed in 5.0 which is not yet released. I will consider making another release but I was hoping not to.

**Comment by fightlikeagirl** [comment]
@linvi thank you so much then I will wait for 5.0 release 👍

**Comment by jxlarrea** [comment]
Hi @linvi, Is this bug still present in the latest 5.0 alpha 6 release?

**Comment by jxlarrea** [comment]
So, @linvi I saw the commit to this bug and I assume the fix is present in version 5.0 alpha6. Unfortunately, there might be some other issue in the library as very few tweets are returned in `MatchingTweetReceived`, most of them show up in `NonMatchingTweetReceived`. Not sure if this is related to the LanguageFilter bug but reverting to version 2.1 fixes the problem.

**Comment by linvi** [comment]
@jxlarrea it should be fixed in 5.0 alpha. I will look into the bug you opened.
